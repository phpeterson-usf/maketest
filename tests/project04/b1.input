-h $(repo_dir)/b1.hex $(TESTS_DIR)/b1.s
