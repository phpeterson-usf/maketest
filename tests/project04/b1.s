main:
    add r0, r1, r2
    b foo
foo:
    add r3, r2, r1
    bx lr
