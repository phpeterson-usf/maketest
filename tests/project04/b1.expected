v2.0 raw
e0810002
eaffffff
e0823001
e12fff1e
